Codeinwp/neve
# Feat/hfg global background [WIP]

### Summary

Added settings for the global header background.

### Will affect visual aspect of the product

NO

### Screenshots 

### Test instructions


- Go to Customize -> Header -> Global
- When the toggle control is off, it should deactivate the background settings for individual rows
- When it's on, the background settings for the global header should not be visible. Instead, 3 shortcuts for the header row background settings should appear.
- The global header background setting should behave normally and keep the changes in the front-end.
- It also should work fine when the Neve Pro Addon is active: the controls from 'Global Header Settings' merge with the new controls.
- The footer, page header, and header sidebar shouldn't be affected in any way.


Closes #3072
